tianshou/highlevel/module/actor.py

```diff
@@ -202,6 +225,9 @@ def create_module(self, envs: Environments, device: TDevice) -> BaseActor:
 
         return actor
 
+    def create_dist_fn(self, envs: Environments) -> TDistFnDiscrOrCont | None:
```

Comment:
Return type can be fixed to `TDistFnCont`

Reply:
See above.